inc/includes/nav-walker.php

```diff
@@ -32,174 +83,519 @@ public function start_lvl( &$output, $depth = 0, $args = array() ) {
       $icon = ob_get_clean();
       $output .= '<button class="dropdown-toggle" aria-expanded="false" aria-label="' . get_default_localization( 'Open child menu' ) . '">';
       $output .= $icon . '</button>';
-      $output .= "\n$indent<ul class=\"sub-menu\">\n";
-    } else {
-      $output .= "\n$indent<ul>\n";
```

Comment:
Why is this removed? There's still need for non-dropdown menus for example in footers.

Reply:
If you look a bit further at line 103 you see it's not removed. It's replaced with new intend and labelledby.

Reply:
Ah, my bad, the diff was confusing.